src/profile/components/Settings/Menu/index.tsx

```diff
@@ -1,21 +1,19 @@
 import React from 'react';
-import { routes } from '..';
+
+import * as appUrls from 'core/appUrls';
+
 import style from './menu.less';
 import Tab from './Tab';
 
-export interface IProps {
-  path: string;
-}
-
-const Menu = ({ path }: IProps) => (
+const Menu = () => (
   <div className={style.menu}>
-    <Tab path={routes.main} active={path === routes.main} text="Informasjon" />
-    <Tab path={routes.mail} active={path === routes.mail} text="E-Post" />
-    <Tab path={routes.password} active={path === routes.password} text="Passord" />
-    <Tab path={routes.penalties} active={path === routes.penalties} text="Prikker og suspensjoner" />
-    <Tab path={routes.privacy} active={path === routes.privacy} text="Personvern" />
-    <Tab path={routes.accessCard} active={path === routes.accessCard} text="Adgangskort (NTNU)" />
-    <Tab path={routes.notifications} active={path === routes.notifications} text="Notifikasjoner" />
+    <Tab text="Informasjon" {...appUrls.getProfileSettingsUrl()} />
```

Comment:
The same question regarding which is current applies here as well I assume

Reply:
I hope the other answer helps here as well :smile: